install/hst-install-ubuntu.sh

```diff
@@ -662,6 +662,9 @@ fi
 # Define apt conf location
 apt=/etc/apt/sources.list.d
 
+# Create new folder if not all-ready exists
+mkdir /root/.gnupg/ && chmod 700 /root/.gnupg/
```

Comment:
use `mkdir -p` , it will check if it already exists and wont throw errors if it does

Reply:
I thought -p is only used when you want to create a full directory path like /root/12/34/56
I added -p to mkdir, thanks!